src/core/queryObserver.ts

```diff
@@ -586,6 +586,7 @@ export class QueryObserver<
       error,
       errorUpdatedAt,
       failureCount: state.fetchFailureCount,
+      errorUpdateCount: state.errorUpdateCount,
```

Comment:
let's maybe name it `errorCount` when exposing, to be in line with `failureCount` ?

Reply:
> don't you think errorCount / failureCount will cause confusion?
> 
> could maybe provide better DOC description for what is errorCount and how is different from failureCount , The sum of all errors could be interpreted to many cases I suppose.
> 
> if not I can proceed with your suggestion :)

Reply:
yeah, we can also keep it at `errorUpdateCount`. If we expose a `query`, e.g. via the predicate filter function, it will also be accessible as:

```
query => query.state.errorUpdateCount
```

the rename might cause confusion, too.